spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowScoreFunction.scala

```diff
@@ -0,0 +1,70 @@
+package org.apache.spot.netflow.model
+
+import org.apache.spark.broadcast.Broadcast
+import org.apache.spot.SuspiciousConnectsScoreFunction
+import org.apache.spot.netflow.{FlowWordCreator, FlowWords}
+
+
+/**
+  * Estimate the probabilities of network events using a [[FlowSuspiciousConnectsModel]]
+  * @param timeCuts
+  * @param ibytCuts
+  * @param ipktCuts
+  * @param topicCount
+  * @param ipToTopicMixBC
+  * @param wordToPerTopicProbBC
+  */
+
+class FlowScoreFunction(timeCuts: Array[Double],
+                        ibytCuts: Array[Double],
+                        ipktCuts: Array[Double],
+                       topicCount: Int,
+                       ipToTopicMixBC: Broadcast[Map[String, Array[Double]]],
+                       wordToPerTopicProbBC: Broadcast[Map[String, Array[Double]]]) extends Serializable {
+
+
+  val suspiciousConnectsScoreFunction =
+    new SuspiciousConnectsScoreFunction(topicCount, ipToTopicMixBC, wordToPerTopicProbBC)
```

Comment:
There should be a specific implementation of ScoreFunction for flow. Right now calling SuspiciousConnectsScoreFunction.score will use broadcast variables and not join.  Even though I have been trying to make this work with Broadcast variables, I haven't been able to do that so we need to keep using join.